test/v2/ThirdPartyRegistry.spec.js

```diff
@@ -3,24 +3,26 @@ import { Mana } from 'decentraland-contract-plugins'
 
 import assertRevert from '../helpers/assertRevert'
 import { balanceSnap } from '../helpers/balanceSnap'
-import {
-  THIRD_PARTY_ITEMS,
-  TIERS,
-  getInitialTiers,
-  ZERO_ADDRESS,
-} from '../helpers/collectionV2'
+import { THIRD_PARTY_ITEMS, ZERO_ADDRESS } from '../helpers/collectionV2'
 import { sendMetaTx } from '../helpers/metaTx'
 
-const Tiers = artifacts.require('Tiers')
 const Committee = artifacts.require('Committee')
 const ThirdPartyRegistry = artifacts.require('ThirdPartyRegistry')
+const ChainlinkOracle = artifacts.require('ChainlinkOracle')
+const InvalidOracle = artifacts.require('DummyInvalidOracle')
+const DummyAggregatorV3Interface = artifacts.require(
+  'DummyAggregatorV3Interface'
+)
 
 const BN = web3.utils.BN
+const toBN = web3.utils.toBN
 const expect = require('chai').use(require('bn-chai')(BN)).expect
 const domain = 'Decentraland Third Party Registry'
 const version = '1'
 const initialValueForThirdParties = true
 const initialValueForItems = false
+const oneEther = toBN('1000000000000000000')
```

Comment:
We can use the web3 utils `web3.utils.toWei()`

Reply:
Done